misc/default.yml

```diff
@@ -30,7 +49,13 @@ platforms:
     gcc:
       features:
       defines:
-        - __AVR_ATmega328__
+        # The due should use the ATSAM3X8E; however, that component does not seem
+        # to be defined for the Godmode tests/components
+        # The next closest part would be the AVR_ATmega2560 as both boards
+        # use the mega form factor
```

Comment:
@Doom4535 can you explain this comment a bit?  I'm not sure why the fix belongs here and not in `Godmode.h`

Reply:
It's been a while since I looked at this, I believe I made that change because the Arduino Duo is most like an Arduino Mega based on pinout. Updating [ `godmode.h`](https://github.com/Arduino-CI/arduino_ci/blob/29ab7b3949e95fb184b00b0809e8b53f1f0b2769/cpp/arduino/Godmode.h#L3-5) to include the equivalent for the SAM board would be the technically correct result, and allow for including support for any extra features the SAM board supports that can't be done with the AVR board.

I'm not sure what the replacement should be, on windows it would likely be something inside of `%LocalAppData%\Arduino15\packages\arduino\hardware\sam`, maybe `%LocalAppData%\Arduino15\packages\arduino\hardware\sam\1.6.12\variants\arduino_due_x\variant.h`?